client/src/declarative/netflux/app.js

```diff
@@ -188,112 +121,221 @@ export default {
             "$schema": "http://json-schema.org/draft-07/schema#",
             "type": "array",
             "maxItems": 0
+        },
+        "search_result_schema": {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "type": "object",
+            "required": ["poster_path", "media_type", "vote_average"],
+            "properties": {
+                "poster_path": {"type": "string", "minLength": 1},
+                "media_type": {"type": "string", "pattern": "^(?:tv|movie)$"},
+                "vote_average": {"type": "number"}
+            }
         }
     },
     "$composers": {
-        MovieRow,
-        ShowRow,
-        SearchRow,
-        "movie_title": {"$compose": "read", "$value": "$.app['$states'].movie.title", "$default": "movie_title"},
-        "movie_overview": {"$compose": "read", "$value": "$.app['$states'].movie.overview", "$default": "movie_overview"},
-        "movie_runtime": [
-            {
-                "$compose": "create",
-                "$value": {"runtime": {"$compose": "read", "$value": "$.app['$states'].movie.runtime", "$default": 0}}
-            },
-            {
-                "$compose": "expand", "$type": "template", "$value": "Runtime: {runtime} minutes ",
-                "$default": "Runtime: {runtime} minutes "
-            }
-        ],
-        "movie_rating": [
-            {
-                "$compose": "create",
-                "$value": {"rating": {"$compose": "read", "$value": "$.app['$states'].movie.rating", "$default": 0}}
-            },
-            {"$compose": "expand", "$type": "template", "$value": "Rating: {rating} ", "$default": "Rating: {rating} "}
-        ],
-        "movie_release": [
-            {
-                "$compose": "create",
-                "$value": {"release": {"$compose": "read", "$value": "$.app['$states'].movie.release", "$default": ""}}
-            },
-            {"$compose": "expand", "$type": "template", "$value": "Release: {release} ", "$default": "Release: {release} "}
-        ],
         "movie_background_image": [
             {
                 "$compose": "create",
                 "$value": {
                     "background_image": {
-                        "$compose": "read", "$value": "$.app['$states'].movie.background_image",
+                        "$compose": "read", "$value": "$.app.$states.movie.background_image",
```

Comment:
I should probably eventually make it so that the application declaration properties are addressable in a normalized way that removes the `$` especially to make JSON Path expressions less ambiguous.